packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -366,13 +366,18 @@ export class ChatContext {
   }
 
   private clearError(targets: ChatErrorTargets[]): void {
```

Comment:
This was creating a spurious `stateChanged` event when there were no errors to clear at all.